helix-view/src/base64/mod.rs

```diff
@@ -0,0 +1,135 @@
+use std::ops::{BitAnd, BitOr, Shl, Shr};
+
+const PAD_BYTE: u8 = b'=';
```

Comment:
It looks like there are some compilation warnings since this ends up not being used on Windows